client/src/components/FlightBarogramm.vue

```diff
@@ -244,8 +157,71 @@ const config: ChartConfiguration<ChartType> = {
   data: {
     datasets: chartData.value,
   },
-  options: options(updatePositionDetails),
+  options: options,
 };
 </script>
+<template>
```

Comment:
Beim neuladen der Seite sind z.B. die Werte für Uhrzeit, Höhe, Wind, Luftdruch nicht geladen... Wenn man dann auf das Baro geht "spingen" einem die neuen Werte dann ins Gesicht.
Beim ersten laden sollte deshalb auch die ersten gültigen Werte bereits geladen werden.

Ich habe selbst mal versucht das nachzuvollziehen, finde aber nicht den richtigen Punkt wo ich das ändern kann. Evlt wäre da auch etwas mehr Doku gut ;-)

Reply:
Das ist etwas komplizierter, da die Werte vom Baro Hover Callback kommen. Hab ich in dem anderen Kommentar jetzt auch erklärt. Würde ich später hinzufügen.